packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -710,7 +713,8 @@ class SemanticsData with Diagnosticable {
   ///
   /// If [label] is not empty, then [textDirection] must also not be null.
   SemanticsData({
-    required this.flags,
```

Comment:
either mark this deprecate or remove it outright, I would try to remove it directly unless it breaks customer test because this is a mostly an internal class that I don't think there is many people interact directly with this. We should still write a courtusy migration guide though

Reply:
removed!